src/main/resources/locales/zh-cn.yml

```diff
@@ -1,33 +1,54 @@
-homes_menu_title: '%1%''的家'
+# 标题
+homes_menu_title: '%1% 的家'
 public_homes_menu_title: '开放的家'
 warps_menu_title: '地标'
+# 编辑页面标题
+home_editor_title: '编辑家: %1%'
+warp_editor_title: '编辑地标: %1%'
+
+# 物品
```

Comment:
Can we remove these comments from the localization files so that it's consistent with all my other plugins? Best keep these internal resources clean and simple as they're shipped in with the jar and won't be seen otherwise.